src/action/linux/start_systemd_unit.rs

```diff
@@ -17,11 +17,27 @@ pub struct StartSystemdUnit {
 impl StartSystemdUnit {
     #[tracing::instrument(level = "debug", skip_all)]
     pub async fn plan(unit: impl AsRef<str>) -> Result<StatefulAction<Self>, ActionError> {
+        let unit = unit.as_ref();
+        // If `systemctl is-active $UNIT` has exit status 3, it doesn't exist
```

Comment:
Does this comment still need to be here? It doesn't look like we worry about "exit status 3" anywhere, so I don't think we need to worry about documenting what it means.

```suggestion
```